### DISCUSSION TITLE
How to enable command in survival mode

### DISCUSSION THREAD

**Discussion by Faultiness** [itzg/docker-minecraft-server#2516] [Q&A]
Hi. I want to use some commands (/give, /gamemode, etc.) in survival mode. I tried to init the container with `-e ALLOW_CHEATS=TRUE`, but it did not work. 

My env: aarch64 with java8-multiarch, version 1.16.5, forge enabled

Thanks for any suggestions.

**Comment by itzg** [comment]
Giving your user "op" might help 

https://docker-minecraft-server.readthedocs.io/en/latest/configuration/server-properties/#opadministrator-players
